File: java/2-conditional-struct/even-or-odd/src/Main.java
// Write a program that asks the user to enter a number and prints whether it is even or odd.

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int number;

        System.out.print("Enter a number: ");
        number = sc.nextInt();

        if (number % 2 == 0) {
            System.out.println("\nThe number entered is even.");
        }
        else {
            System.out.println("\nThe number entered is odd.");
        }
    }
}

File: c/2-conditional-struct/even-or-odd/main.c
// Write a program that asks the user to enter a number and prints whether it is even or odd.

#include <stdio.h>
#include <stdlib.h>

int main()
{
    int number;

    printf("Enter a number: ");
    scanf("%d", &number);

    if (number % 2 == 0) {
        printf("\nThe number entered is even.\n");
    }
    else {
        printf("\nThe number entered is odd.\n");
    }

    return 0;
}
